.github/workflows/test_openvino.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             "*training*",
             "*export*",
           ]
-        transformers-version: ["4.36.0", "latest"]
```

Comment:
I'm not at all against dropping v4.36 support, but would keep tests for the lowest "official" transformers version supported (we can an additional check for this), would that work ?

Reply:
Thanks Ella! Agreed, we should definitely keep testing the lowest supported version. I hope 4.37 works.

Reply:
@AlexKoff88 Yes, testing now, I will create a new PR when everything works (because some minor changes are needed). Looks like 4.37 will work.